Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon_test.py

```diff
@@ -2823,3 +2832,136 @@ def test_get_endpint_command(requests_mock, mocker):
+    requests_mock.post(
+        f'{SERVER_URL}/devices/entities/host-groups/v1',
+        json=response_data,
+        status_code=400,
+        reason='Bad Request'
+    )
+    with pytest.raises(SystemExit):
+        create_host_group_command(name="dem test",
+                                  description="dem des",
+                                  group_type='static',
+                                  assignment_rule="device_id:[''],hostname:['falcon-crowdstrike-sensor-centos7']")
+
+
+def test_update_hostgroup_invalid(requests_mock):
+    """
+    Test Create hostgroup with valid args with unsuccessful args
+    Given
+     - Invalid arguments for hostgroup
+    When
+     - Calling create hostgroup command
+    Then
+     - Throw an error
+     """
+    from CrowdStrikeFalcon import update_host_group_command
+    response_data = load_json('test_data/test_create_hostgroup_invalid_data.json')
+    requests_mock.patch(
+        f'{SERVER_URL}/devices/entities/host-groups/v1',
+        json=response_data,
+        status_code=400,
+        reason='Bad Request'
+    )
+    with pytest.raises(SystemExit):
+        update_host_group_command(
+            host_group_id='b1a0cd73ecab411581cbe467fc3319f5',
+            name="dem test",
+            description="dem des",
+            group_type='static',
+            assignment_rule="device_id:[''],hostname:['falcon-crowdstrike-sensor-centos7']")
+
+
+@pytest.mark.parametrize('status, expected_status_api', [('New', "20"),
+                                                         ('Reopened', "25"),
+                                                         ('In Progress', "30"),
+                                                         ('Closed', "40")])
+def test_resolve_incidents(requests_mock, status, expected_status_api):
+    """
+    Test Create resolve incidents with valid status code
+    Given
+     - Valid status, as expected by product description
+    When
+     - Calling resolve incident command
+    Then
+     - Map the status to the status number that the api expects
+     """
+    from CrowdStrikeFalcon import resolve_incident_command
+    m = requests_mock.post(
+        f'{SERVER_URL}/incidents/entities/incident-actions/v1',
+        json={})
+    resolve_incident_command(['test'], status)
+    assert m.last_request.json()['action_parameters'][0]['value'] == expected_status_api
+
+
+def test_resolve_incident_invalid():
+    """
+    Test Create resolve incidents with invalid status code
+    Given
+     - Invalid status, which is not expected by product description
+    When
+     - Calling resolve incident command
+    Then
+     - Throw an error
+     """
+    from CrowdStrikeFalcon import resolve_incident_command
+    with pytest.raises(DemistoException):
+        resolve_incident_command(['test'], '')
+        resolve_incident_command(['test'], 'new')
+        resolve_incident_command(['test'], 'BAD ARG')
```

Comment:
Which call should raise the error?

Reply:
All of them, my mistake here. replaced with `parametrize`